bosh/report.go

```diff
@@ -110,6 +112,27 @@ func (r report) AddJob(jobSpec spec) {
 	r.Host.Nodes[hostID] = node
 }
 
+func (r report) hostId() string {
+	return fmt.Sprintf("%s;<host>", r.hostname)
+}
+
+func (r report) SetMonitProcesses(services []monit.Service) {
+	hostId := r.hostId()
+	node := r.Host.Nodes[hostId]
+
+	for k := range node.Latest {
```

Comment:
I guess we have to delete those entries because this report is long-lived and being rendered in subsequent requests which was fine prior to this since the information was always static and din't change. Now that's different. An alternative to cleaning up here, could have been to create a copy of the report for every request.